### ISSUE TITLE
allow install from a binary

### ISSUE THREAD

**Issue by afeld** [18F/cf-service-connect#7]
1. [x] Create a Release with binaries or various platforms - [here's](https://github.com/18F/cf-route-lookup/blob/master/CONTRIBUTING.md#releasing) the strategy I used most recently
1. [x] Update the README
1. [ ] [Submit to the Cloud Foundry plugin list](https://github.com/cloudfoundry-incubator/cli-plugin-repo#submitting-plugins)

**Comment by afeld** [comment]
Waiting on getting feedback on https://github.com/18F/cf-db-connect/pull/22 before proceeding.

**Comment by afeld** [comment]
Now blocked by #13. Per [the plugin repo](https://github.com/cloudfoundry-incubator/cli-plugin-repo#cross-compile-to-the-3-different-operating-systems):

> at a minimum we want plugins to support linux64, **win64** and osx.

**Comment by afeld** [comment]
Closing in favor of #33.
